build-tools/create-packs/ConfigureLocalWorkload.targets

```diff
@@ -90,20 +62,15 @@
         Targets="_GenerateXAWorkloadContent"
     />
 
-    <GetAndroidWorkloadExtends JsonFilePath="$(_LocalAndroidManifestFolder)WorkloadManifest.json">
-      <Output TaskParameter="ExtendsElement" PropertyName="AndroidWorkloadExtendsElement" />
-    </GetAndroidWorkloadExtends>
-
     <PropertyGroup>
       <_EmptyWorkloadJsonContent>
 <![CDATA[
-{"version": "0.0.1", "workloads": { "android-deps": { "extends" : [ $(AndroidWorkloadExtendsElement) ] } } }
+{"version": "0.0.1", "workloads": { "android-deps": { "extends" : [ "microsoft-net-runtime-android" ] } } }
```

Comment:
Previously, this was installing `microsoft-net-runtime-android-aot`:
```
Installing pack Microsoft.NETCore.App.Runtime.AOT.linux-x64.Cross.android-x86 version 10.0.0-preview.6.25304.106...
```

Which failed on Github actions on linux.

I think we only need `"microsoft-net-runtime-android"` here.